app/views/external_users/claims/basic_fees/_basic_fee_fields.html.haml

```diff
@@ -1,20 +1,50 @@
-%fieldset
-  .form-group.basic-fee-group.nested-fields.fee-details.js-block{class: to_slug(f.object.description), data:{type:'gradFees', autovat: @claim.apply_vat? ? "true" : "false", "block-type": f.object.calculated? ? "FeeBlockCalculator" : "FeeBlock"}}
-    = f.hidden_field :fee_type_id, value: f.object.fee_type_id
-
-    - if f.object.calculated?
-      = render partial: 'external_users/claims/basic_fees/basic_fee_calculated_fields', locals: { f: f, fee: present(f.object) }
-    - else
-      = render partial: 'external_users/claims/basic_fees/basic_fee_uncalculated_fields', locals: { f: f, fee: present(f.object) }
-
-    - if f.object.fee_type.requires_dates_attended?
-      .grid-row.fee-dates-row
-        %label.form-label Dates
-
-        %span.no-dates.form-hint
-          = t('.no_dates')
-        = link_to_add_association t('.add_date_attended'), f, :dates_attended, class: '', partial: 'date_attended_fields_condensed', data: {'association-insertion-method' => 'prepend', 'association-insertion-traversal' => 'next', 'association-insertion-node' => 'div'}
-
-        .grid-row.dates-wrapper
-          = f.fields_for :dates_attended do |date_attended|
-            = render 'date_attended_fields_condensed', { f: date_attended, submodel_count: date_attended.index+1,  parent_model_prefix: "basic_fee_#{@basic_fee_count}" }
+-# Check box input
+.multiple-choice{"data-target" => to_slug(f.object.description)}
+
+  = f.check_box(:toggle, checked: !f.object.blank?, id: "#{to_slug(f.object.description)}_input", "aria-controls" => to_slug(f.object.description), :name => "graduated_fees_checklist")
+
+  %label{:for => "#{to_slug(f.object.description)}_input"}
+    = f.object.description
+
+-# Panel content
+.panel.panel-border-narrow.js-hidden{:id => to_slug(f.object.description)}
+  = f.hidden_field :fee_type_id, value: f.object.fee_type_id
+
+  -# Number of days
+  = f.adp_text_field :quantity, label: t(".#{f.object.fee_type_code.downcase}.quantity"), hint_text:  t(".#{f.object.fee_type_code.downcase}.hint"),
+                        input_classes:'quantity form-control-1-8',
+                        input_type: 'number',
+                        value: present(f.object).quantity,
+                        errors: @error_presenter
+  -# Rate
+  = f.adp_text_field :rate,
+                        label: t(".rate"),
+                        input_classes:'rate form-control-1-4 form-input-denote__input',
+                        input_type: 'currency',
+                        errors: @error_presenter,
+                        value: present(f.object).rate
+
```

Comment:
This rate field ^^ is now back in the page.

The value is being formatted and includes a `£0.00` - you might see an error in the console:

```
> The specified value "\u00A30.00" is not a valid number.
```

@lostie